suzieq/poller/worker/nodes/node.py

```diff
@@ -1454,7 +1456,7 @@ async def _init_ssh(self, init_dev_data=True,
 
                 output = await self.wait_for_prompt()
                 if output.strip().endswith('>'):
-                    if not self._handle_privilege_escalation():
+                    if await self._handle_privilege_escalation():
```

Comment:
Can we change this check to `if await self._handle_privilege_escalation() == -1`? It's easier to read

Reply:
OK, I'm used to the Unix way of thinking :)